site/docs/src/json/tenants/request.json

```diff
@@ -1,5 +1,6 @@
 {
   "tenant": {
+    "configured": true,
```

Comment:
I don't think this is a mutable value, so it would not be documented as part of the request.

Reply:
Hmm, didn't realize that this was a flag for mutation.